tests/unit/Cache/Backend/RedisCest.php

```diff
@@ -44,11 +44,15 @@ public function increment(UnitTester $I)
         $I->wantTo('Increment counter by using Redis as cache backend');
 
         $key = '_PHCR' . 'increment';
-        $cache = new Redis(new Data(['lifetime' => 20]), [
-            'host'  => env('TEST_RS_HOST', '127.0.0.1'),
-            'port'  => env('TEST_RS_PORT', 6379),
-            'index' => env('TEST_RS_DB', 0),
-        ]);
+        $cache = new Redis(
+            new Data(['lifetime' => 20]),
+            [
+                'host'     => env('TEST_RS_HOST', '127.0.0.1'),
+                'port'     => env('TEST_RS_PORT', 6379),
+                'index'    => env('TEST_RS_DB', 0),
+                'statsKey' => '_PHCR',
```

Comment:
Well, all these changes will now require the user to pass `statsKey` key more often, but previously not.

I think from the point of view of the Phalcon user this changelog entry "Made the `specialKey` (`_PHCR`) optional" is wrong because now he is forced to play with this key almost always (or at least more often)

Reply:
I disagree. The reason all these `statsKey` elements appear in the tests is because we assumed that the key would always be there. So all the existing tests must have that in the initialization of the `Redis` component. 

For an real life application, this initialization would happen mostly in the DI container and there will be no more changes.